conu/apidefs/container.py

```diff
@@ -253,3 +253,11 @@ def get_file(self, file_path):
         """
         raise NotImplementedError("get_file method is not implemented")
 
+    def check_file_exists(self, filename):
```

Comment:
What is the expected behavior when we pass path to directory? It also seems that `filename` actually represents path to a file -- looking at `os.path` module, they usually name an argument like this `path`.